### PR TITLE
Spelling: No global, preventiong translations, for uploaded translation,no disabled

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: weblate/formats/ttkit.py [hunk 1]
```diff
@@ -670,7 +670,7 @@ def target(self):
 
 
 class PoFormat(TTKitFormat):
-    name = _('Gettext PO file')
+    name = _('gettext PO file')
```
Comment (nijel) [inline weblate/formats/ttkit.py]: These are labels for the file format, all are starting with uppercase, so I'd rather keep this consistent.

File: weblate/lang/models.py [hunk 2]
```diff
@@ -523,7 +523,7 @@ class Plural(models.Model):
         verbose_name=ugettext_lazy('Plural definition source'),
         choices=(
             (SOURCE_DEFAULT, ugettext_lazy('Default plural')),
-            (SOURCE_GETTEXT, ugettext_lazy('Gettext plural formula')),
+            (SOURCE_GETTEXT, ugettext_lazy('Plural gettext formula')),
```
Comment (nijel) [inline weblate/lang/models.py]: It's plural formula coming from gettext, so I believe the original string was correct.

File: weblate/templates/dictionary.html [hunk 3]
```diff
@@ -31,7 +31,7 @@
     </a>
     <ul class="dropdown-menu">
 <li><a href="{% url 'download_dictionary' project=project.slug lang=language.code %}?format=csv">{% trans "Comma separated values (CSV)" %}</a></li>
-<li><a href="{% url 'download_dictionary' project=project.slug lang=language.code %}?format=po">{% trans "Gettext (PO)" %}</a></li>
+<li><a href="{% url 'download_dictionary' project=project.slug lang=language.code %}?format=po">{% trans "gettext (PO)" %}</a></li>
```
Comment (nijel) [inline weblate/templates/dictionary.html]: Same as above - "Comma separated" is with capital letter, same should be done for Gettext.

File: weblate/lang/models.py (hunk 2 above)
Comment (comradekingu) [reply weblate/lang/models.py]: In my head that is a "Plural gettext formula", I could be wrong.

File: weblate/templates/dictionary.html (hunk 3 above)
Comment (comradekingu) [reply weblate/templates/dictionary.html]: gettext supports MO and other things too, no?

File: weblate/formats/ttkit.py (hunk 1 above)
Comment (comradekingu) [reply weblate/formats/ttkit.py]: Hmmm, I figured it was easier to always use it like upstream gettext uses it.
Is it a solution to go "GNU gettext"?
Also, I've seen "i18next", which isn't "I18next"

Comment (nijel) [reply weblate/formats/ttkit.py]: True, we should be consistent with i18next as well. However even upstream is not consistent on capitalization here when used as first word in a sentence, see https://www.i18next.com/. But in most case they also use lowercase...

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**